artifactory/commands/utils/result.go

```diff
@@ -42,7 +42,7 @@ func (r *Result) SetReader(reader *content.ContentReader) {
 
 // Reads and pars Deployed artifacts details from a given file.
```

Comment:
// Reads and pars Deployed artifacts details from a given file.
-->
// Reads and parses the deployed artifacts details from the provided file.